packages/tracing-processing/src/models/models.ts

```
@@ -0,0 +1,47 @@
+import { S3BodySchema } from "pagopa-interop-tracing-models";
+import { decodeSQSMessageError } from "./errors.js";
+import { SQS } from "pagopa-interop-tracing-commons";
+import { TracingFromS3Path } from "./tracing.js";
+
+export function decodeSqsMessage(message: SQS.Message): TracingFromS3Path {
+  try {
+    const messageBody = message.Body;
+
+    if (!messageBody) {
+      throw "Message body is undefined";
+    }
+    const s3Body = S3BodySchema.safeParse(JSON.parse(messageBody));
```

is this parse required? IMO it's too detailed on fields we cannot control.
we could just use the type in the library and fail if one of necessary fields is missing.